.github/workflows/tests.yml

```diff
@@ -0,0 +1,37 @@
+name: tests
+on: [pull_request]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.5.7, 2.6.5, 2.7.0]
```

Comment:
We should make some updates here, like I did with Puma Pool Usage. At minimum, there should be something in the Ruby 3.x line. I think:

`ruby: [2.7.6, 3.0.4, 3.1.2]`

If you make the minimum 2.7.6, you can remove `gem install bundler`, `gem update`, and `bundle install`, which would happen automatically.